File: 2006Scape Server/src/main/java/com/rs2/integrations/discord/commands/Vote.java
package com.rs2.integrations.discord.commands;

import com.rs2.Constants;
import org.javacord.api.entity.message.Message;
import org.javacord.api.event.message.MessageCreateEvent;
import org.javacord.api.listener.message.MessageCreateListener;

public class Vote implements MessageCreateListener {
    @Override
    public void onMessageCreate(MessageCreateEvent event) {
        Message message = event.getMessage();
        if (message.getContent().equalsIgnoreCase("::vote")) {
            if (Constants.WORLD == 1) {
                event.getChannel().sendMessage("Visit " + Constants.WEBSITE_LINK + "/vote.html then type \"::claimvote\" in-game to receive your reward!");
            }
        }
    }
}

File: 2006Scape Server/src/main/java/com/rs2/integrations/discord/commands/Vote.kt
package com.rs2.integrations.discord.commands

import com.rs2.GameConstants
import org.javacord.api.event.message.MessageCreateEvent
import org.javacord.api.listener.message.MessageCreateListener

class Vote : MessageCreateListener {
    override fun onMessageCreate(event: MessageCreateEvent) {
        val message = event.message
        if (message.content.equals("::vote", ignoreCase = true)) {
            if (GameConstants.WORLD == 1) {
                event.channel.sendMessage("Visit " + GameConstants.WEBSITE_LINK + "/vote.html then type \"::claimvote\" in-game to receive your reward!")
            }
        }
    }
}